resources_portal/models/user_notification.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from .user import User
+
+
+class UserNotification(models.Model):
+    class Meta:
```

Comment:
Can you add a docstring explaining this model? I actually forget exactly how we intended to use it.

Reply:
Same here, I added it because I saw it on the original sketch. I'm guessing we'll send a notification to users when they need to act on something? Sign, review, etc?

Reply:
Seems like something we should figure out then! My guess is that we were going to have this table be for notification settings. For each type of event that we'd notify about we would have a `event_type` choice for. Each user can choose whether or not they want notifications for that type of event for each organization. We can create a default set for each user for each organization they create/join, then delete some of them or create more as the user tweaks their preferences on a per-organization basis. I think that's why there was an `(Org)` written in front of the table name.

I think the assigned_to field was about whether they wanted notifications for all events of that type or just the ones where they were the assignee.

Does that sound right? CC @davidsmejia @dvenprasad 

Also @dvenprasad do you have a list of notification `event_type`s we can use for the choices?

Reply:
Yep that sounds about right from what I can recall. 

I don't have a full list of all the `event_types`. Here are the notifications we'll need to send as far as requests are concerned. 
- New Resource Request
- Change in Request status
- Request Approved/Rejected
- Assigned to a request
- Reminders to take actions: Review a request, sign and upload MTAs

Are these sufficient for now? 
There will be more as we dive into each of the modules.

Reply:
Yeah that's great! Thanks Deepa!

@arielsvn, can you add an Organization to this table, and then also add those `event_types` as choices?

Reply:
should we have a cancelled state?

Reply:
@kurtwheeler I'm having issues understanding this model. Why should we have a different row for each setting instead of a single one for all of them? We could also have a model like `OrganizationUserNotificationSettings`, and add one field for each type of setting. Example:

```py
new_resource_request = models.BooleanField(default=True)
change_request_status = models.BooleanField(default=False)
assignments = models.BooleanField(default=True)
# ... etc
```

@dvenprasad do you have any designs, even lo-fi, of how the users will be able to modify and view these notifications?

Reply:
We talked and decided to rename the model to `OrganizationUserSettings` to be able to store more general settings. I'll leave a comment on the model describing how to add the settings, we can add them once we start working on that feature.